docs/topics/limits.mdx

```diff
@@ -32,7 +32,7 @@ Service quotas are directly affected by the support plan on your account. Both <
     <tr>
       <td>Operations per second</td>
       <td>namespace</td>
-      <td>100</td>
+      <td>10</td>
       <td>1,000</td>
       <td>10,000</td>
```

Comment:
these need to be updated to be 10x less too because the tiering is always 10x the previous tier's limit

Reply:
done